src/contexts/FilterSelectContext.tsx

```
@@ -0,0 +1,71 @@
+      setSelectedYear(yearSearchParam)
+      return
+    }
+
+    // Invalid year in URL - reset to default and clean URL
+    setSelectedYear(latestYear)
+    setSearchParams({}, { replace: true })
+  }, [searchParams, setSearchParams])
+
+  const updateSelectedYear = useCallback(
+    (year: string) => {
+      if (!isValidYear(year)) return
```

ESLint

[error] Expected { after 'if' condition.

(curly)